espnet2/asr/frontend/sinc.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+#  2020, Technische Universität München;  Ludwig Kürzinger
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+"""Sinc convolutions for raw audio input."""
+
+from collections import OrderedDict
+from espnet2.asr.frontend.abs_frontend import AbsFrontend
+from espnet2.layers.sinc_conv import LogCompression
+from espnet2.layers.sinc_conv import SincConv
+import humanfriendly
+import torch
+from typing import Tuple
+from typing import Union
+
+
+class LightweightSincConvs(AbsFrontend):
+    """Lightweight Sinc Convolutions.
+
+    Provide a frontend for raw audio input.
+    https://arxiv.org/abs/2010.07597
+    """
+
+    def __init__(
+        self,
+        fs: Union[int, str] = 16000,
+        in_channels=1,
```

Comment:
please add type annotation for all functions.

Reply:
OK